general/1_terminal/Dima_Mayakov/Solution_task_2.txt

```diff
@@ -0,0 +1,5 @@
+It's all about the flow of execution of commands
+ and the transfer of control between them.
+ The script runs in its own thread and cannot affect the thread in which the terminal is running to change the directory
```

Comment:
Not really a thread, but the whole different subprocess, but you got the idea right.